spec.html

```diff
@@ -39767,6 +39966,27 @@ <h1>
         </emu-alg>
       </emu-clause>
 
+      <emu-clause id="typedarray-create-same-type" type="abstract operation">
+        <h1>
+          TypedArrayCreateSameType (
+            _exemplar_: a TypedArray,
```

Comment:
```suggestion
            _exemplar_: a Typed Array,
```

Reply:
Hm? The status quo has 20 occurrences of "a TypedArray", none of "a Typed Array".

Reply:
hm, ok, i didn't check but it looked wrong